src/lib/db/api-token-store.ts

```diff
@@ -96,12 +96,26 @@ export class ApiTokenStore implements IApiTokenStore {
             });
     }
 
-    count(): Promise<number> {
+    async count(): Promise<number> {
         return this.db(TABLE)
             .count('*')
             .then((res) => Number(res[0].count));
     }
 
+    async countByType(): Promise<Map<string, number>> {
+        return this.db(TABLE)
+            .select('type')
+            .count('*')
+            .groupBy('type')
+            .then((res) => {
```

Comment:
We should try to use `async` / `await` instead of `then` when possible.

Reply:
In general I agree. This was to follow the semantics in the file.